doc/release/0.19.0-notes.rst

```diff
@@ -65,6 +65,12 @@ repeatable minimizations.
 The keyword `sigma` in `scipy.optimize.curve_fit` was overloaded to also accept
 the covariance matrix of errors in the data.
 
+The method `trust-region-exact` was added to the function `scipy.optimize.minimize`.
+This new trust-region method solves the subproblem with higher accuracy at the cost
+of more hessian factorizations (compared to dogleg) and is able to deal with indefinite
```

Comment:
Let's write it as "Hessian" with the capital letter for consistency. 

Also I believe you need to use an article, as I checked it is usually written as "the Hessian" ("Hessian factorizations" sounds correct though). I guess you can also write "with indefinite Hessians". Choose what you like and maybe check other places.